api/src/pcapi/core/educational/repository.py

```diff
@@ -1321,3 +1321,21 @@ def fetch_venue_for_new_offer(venue_id: int, requested_provider_id: int) -> offe
     if not venue:
         raise offerers_exceptions.VenueNotFoundException()
     return typing.cast(offerers_models.Venue, venue)
+
+
+def has_collective_offers_for_program_and_venue_ids(program_name: str, venue_ids: typing.Iterable[str]) -> bool:
+    query = (
+        educational_models.CollectiveOffer.query.join(
+            educational_models.EducationalInstitution, educational_models.CollectiveOffer.institution
+        )
+        .join(educational_models.EducationalInstitutionProgram, educational_models.EducationalInstitution.programs)
+        .filter(
+            sa.and_(
```

Comment:
Détail : le `and_` n'est pas nécessaire car les arguments de `filter` sont associés en `AND`.